styles.css

```diff
@@ -1,785 +1,871 @@
-      background-position-y: 3vw;
-      background-repeat: no-repeat;
-    }
-    .contact-page{
-      width: 75vw;
-      margin-left: 6vw;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-      font-weight: 400;
-      font-size: 5.579vw;
-      line-height: 6vw;
-      text-align: center;
-      color: #EBEBFF;
-    }
-    .bottomback{
-      width: 80vw;
-      background-image: url("contact-bottom.png");
-      background-size: 20vw 15vw;
-      background-position-x: 60vw;
-      background-position-y: 99.5%;
-      background-repeat: no-repeat;
-    }
-    #name{ 
-      width: 67.5vw;
-      height: 8vw;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-      font-weight: 400;
-      font-size: 4.53vw;
-      line-height: 5vw;
-      display: flex;
-      align-items: center;
-      color: #172B4D;
-      margin-left: 7vw;
-      background: #FFFFFF;
-      border: 1px solid #CFD8DC;
-      border-radius: 8px;
-      padding-left: 5vw;
-      padding-top: 2vw;
-    }
-    #email{
-      width: 67.5vw;
-      height: 8vw;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-      font-weight: 400;
-      font-size: 4.53vw;
-      line-height: 5vw;
-      display: flex;
-      align-items: center;
-      color: #172B4D;
-      margin-left: 7vw;
-      background: #FFFFFF;
-      border: 1px solid #CFD8DC;
-      border-radius: 8px;
-      padding-left: 5vw;
-      padding-top: 2vw;
-      margin-top: 2.5vw;
-    }
-#message{
-      width: 67.5vw;
-      height: 35vw;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-      font-weight: 400;
-      font-size: 4vw;
-      line-height: 4vw;
-      color: #B3BAC5;
-      margin-left: 7vw;
-      background: #FFFFFF;
-      border: 1px solid #CFD8DC;
-      border-radius: 8px;
-      padding-left: 5vw;
-      padding-top: 2vw;
-      margin-top: 2.5vw;
-    }
-    .touch{
-      width: 34vw;
-      height: 10vw;
-      font-family: 'Poppins', sans-serif;
-      font-style: normal;
-      font-weight: 500;
-      font-size: 4.53vw;
-      line-height: 8vw;
-      display: flex;
-      align-items: center;
-      text-align: center;
-      letter-spacing: 0.03em;
-      color: #396DF2;
-      flex: none;
-      order: 0;
-      flex-grow: 0;
-      border: #396DF2 solid 1.5px;
-      background-color: white;
-      border-radius: 8px;
-      padding: 1vw 2vw 1vw 2vw;
-      cursor: pointer;
-      margin-top: 2.5vw;
-      margin-left: 7vw;
-    }
-    .touch:hover{background-color: #396DF2; border:  #FFFFFF solid 2px;color:  #FFFFFF;}
-    .touch:active{background-color: #FFFFFF;color: #000000;border:  #FFFFFF solid 2px;}
-    .foot{
-      width: 27.5vw;
-      height: 1vw;
-      margin-left: 30vw;
-      margin-bottom: 2vw;
-      background: #0D151E;
-      border-radius: 10px;
-      margin-top: 7.5vw;
-    }
\ No newline at end of file
+@media only screen and (min-width: 768px) {

```

Comment:
- [ ] You have done great so far. :+1: :+1: You have implemented most of the changes suggested by the previous reviewer, But there is one major requirement you missed.
- [ ] As pointed out by the previous reviewer, your portfolio needs to be responsive and look good both on desktop and mobile screens. Currently, the way your CSS is structured, the media query applies the styles to screens of `768px` and above, which makes the Mobile screens appear without styling. Following the best practice, the best way to write CSS is following the mobile-first rule where you write CSS for mobile screens and then add Media query to change the look of some parts on desktop screens.
- - I'll suggest you follow the suggestion above and update your CSS so that your style rules can apply to both Mobile and Desktop screens. :smile: 

 - - VIDEO REFERENCE:

https://github.com/feyisakedir/Portfolio-2nd-Project/assets/107632780/c2bc782a-b80f-4f51-ab64-9e69cc66d85c

- [OPTIONAL] I'll suggest you create an `image or assets` folder and move your images into that folder, It will make your file structure and code base more organized. :+1: :+1: